src/proxy-good-queue.js

```diff
@@ -0,0 +1,508 @@
+                    //send pool for login
+                    sendToPool(usersConnections[id_user], {
+                        id: rpcId,
+                        method: "login",
+                        params: {
+                        login: login,
+                        pass: usersConnections[id_user].options.pass || "x"
+                    }
+                    });
+                break; }
+                
+                    //send data pool
+                    case "submit": { 
+					console.log("SUBMIT =================>");
+					console.log("SUBMIT =================>");
+					console.log("SUBMIT =================>");
+					console.log("SUBMIT =================>");
+					console.log("SUBMIT =================>");
+					console.log("SUBMIT =================>");
+					console.log("SUBMIT =================>");
+					console.log("SUBMIT =================>");
+					console.log("SUBMIT =================>");
+                        sendToPool(usersConnections[id_user], {
+                            id: getRpcId(usersConnections[id_user]),
+                            method: "submit",
+                            params: {
+                                id: addressConections[usersConnections[id_user].address].workerId[usersConnections[id_user].id_user],
+                                job_id: data.params.job_id,
+                                nonce: data.params.nonce,
+                                result: data.params.result
+                            }
+                        }); 
+                    break;
+                    }
+                }
+          });
+        
+        
+        //Init Socket
+        addressConections[address].socket.connect(+connection.options.port, connection.options.host, function() {
+        
+       
+            
+            //set online user, Hashes
+            connection.online = true;
+            connection.hashes = 1;
+          
+            //set on data
+            addressConections[address].socket.on("data", function(chunk) {
+                
+                //save data recived in buffer
+                addressConections[address].buffer += chunk;
+              
+                //Checking has buffer
+                while(addressConections[address].buffer && addressConections[address].buffer.includes("\n")) {
+                    
+                    
+                    //get end line
+                    const newLineIndex = addressConections[address].buffer.indexOf("\n");
+                    
+                    //get mensaje
+                    const stratumMessage = addressConections[address].buffer.slice(0, newLineIndex);
+                    
+                    //remove line from buffer
+                    addressConections[address].buffer = addressConections[address].buffer.slice(newLineIndex + 1);
+          
+                    //ad 1 user
+
+                    
+                    log("[ADDRESS]["+addressConections[address].users+"] "+ address);
```

Comment:
````js
log(`[ADDRESS][${addressConections[address].users}] ${address}`);
````

Reply:
proxy-good-queue.js ignores that file, it works best with proxy.js